### DISCUSSION TITLE
Update desktop app

### DISCUSSION THREAD

**Discussion by igmarketing** [ztjhz/BetterChatGPT#526] [General]
Hi,

Is it possible to update the desktop version (Windows) to include the recent GPT-4 turbo models?

Thank you!!

**Comment by maphew** [comment]
For me the new models are already available under _Settings >> default chat config_

![image](https://github.com/ztjhz/BetterChatGPT/assets/486200/c29e28f6-3ece-4657-bc37-2bf1217266b5)

**Comment by ztjhz** [comment]
@igmarketing The desktop app has just been updated.
